packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -481,38 +482,47 @@ describe('Grant Resolvers', (): void => {
     })
 
     test('Returns error for unknown grant', async (): Promise<void> => {
-      const gqlQuery = appContainer.apolloClient
-        .mutate({
-          mutation: gql`
-            query GetGrant($id: String!) {
-              grant(id: $id) {
-                id
-                client
-                state
-                access {
+      let gqlQuery
+      try {
+        gqlQuery = appContainer.apolloClient
+          .mutate({
+            mutation: gql`
+              query GetGrant($id: ID!) {
+                grant(id: $id) {
                   id
-                  identifier
+                  client
+                  state
+                  access {
+                    id
+                    identifier
+                    createdAt
+                    actions
+                    type
+                  }
                   createdAt
-                  actions
-                  type
                 }
-                createdAt
               }
+            `,
+            variables: {
+              id: uuid()
             }
-          `,
-          variables: {
-            id: uuid()
-          }
-        })
-        .then((query): Grant => {
-          if (query.data) {
-            return query.data.grant
-          } else {
-            throw new Error('Data was empty')
-          }
-        })
+          })
+          .then((query): Grant => {
+            if (query.data) {
+              return query.data.grant
+            } else {
+              throw new Error('Data was empty')
+            }
+          })
 
-      await expect(gqlQuery).rejects.toThrow(ApolloError)
+        await gqlQuery // throw error to inspect error object
+      } catch (err) {
+        await expect(gqlQuery).rejects.toThrow(ApolloError)
+        await expect(gqlQuery).rejects.toThrow('No grant')
```

Comment:
```suggestion
        assert.ok(error instanceof ApolloError)
        expect(error.message).toBe('No grant')
```
since we already have the error & to not need to make the gqlQuery request multiple times